waku/v2/protocol/waku_store/waku_store.nim

```diff
@@ -619,7 +619,7 @@ proc resume*(ws: WakuStore, peerList: Option[seq[RemotePeerInfo]] = none(seq[Rem
     # if no peerList is set then query from one of the peers stored in the peer manager 
     let peerOpt = ws.peerManager.selectPeer(WakuStoreCodec)
     if peerOpt.isNone():
-      error "no suitable remote peers"
+      warn "no suitable remote peers"
```

Comment:
Should this not actually be an error, because at this point, resuming will fail?
Maybe the error message should also say something about resume having failed.

If we want to reserve ERR for things that we cannot recover from, we should also consider lines 622 and 648 for warnings instead of errors.

As @LNSD pointed out, we should agree on which severity level justifies and error.

Reply:
Resuming will indeed fail, but in this case resuming itself failing is not _necessarily_ an error. It simply indicates that the store node has no interest in resuming history from a peer (and many/most store nodes are currently configured in this way). In other words, many correctly configured nwaku nodes will "fail" to resume because they have simply not configured a store peer, indicating that they do not care about resuming past history. I agree that there's probably a better way to handle this (e.g. in configuration), but this change is aimed at keeping ERROR logs from appearing when a correctly-configured node is started up.

Reply:
Thank you for carifying :).